playground/src/utils/helper_functions.tsx

```diff
@@ -177,7 +177,6 @@ export function formatMemory(memory: BigUint64Array): string {
   let output = "";
   for (let i = 0; i < memory.length; i += 5) {
     const memorySlice = memory.slice(i, i + 5);
-    output += `[${memorySlice[0]}]: [${memorySlice[1]}, ${memorySlice[2]}, ${memorySlice[3]}, ${memorySlice[4]}] \n                    `;
+    output += `[${memorySlice[0]}]: [${memorySlice[1]}, ${memorySlice[2]}, ${memorySlice[3]}, ${memorySlice[4]}] \n           `;
```

Comment:
what is the rest of the space for?

Reply:
It's for alignment of the different memory slots